mbuild/formats/gomc_conf_writer.py

```diff
@@ -0,0 +1,4065 @@
+
+        required_ensemble_files_List =  simulation_settings
+
+    elif ensemble_type in ['GEMC_NVT', 'GEMC_NPT', 'GCMC']:
+
+        simulation_settings = ["charmm_object", "ensemble_type", "RunSteps", "Temperature"]
+
+        required_ensemble_files_List =   simulation_settings
+
+
+    else:
+        print('ERROR: The ensemble_type selected for the _get_required_ensemble_files function is not valid.')
+        required_ensemble_files_List = None
+
+    return required_ensemble_files_List
+
+
+
+
+class GOMCControl():
+    def __init__(self, charmm_object, ensemble_type, RunSteps, Temperature, input_variables_dict = None
+                 ):
+
+        """
+
+        Construct the GOMC configuration input file with the defaults,
+        or adding additional data in the input_variable section
+        Default setting for the GOMC configuraion files are based upon the
+        a educated guess which should result in reasonable sampling for a
+        given ensemble/simulation type. However, there is no guarantee that
+        the default setting will provide the best or adequate sampling for
+        the selected system. The user has the option to modify the
+        configuration/contorl files based on the simulation specifics or in to
+        optimize the system beyond the standard settings.  These override
+        options are available via the keyword arguments in input_variable_dict.
+        Parameters
+        ----------
+        charmm_object :  Charmm object, which by definition has been parameterized ' \
+                      + 'from the selected force field.',
+        ensemble_typ : str, only accepts 'NVT', 'NPT', 'GEMC_NPT', 'GCMC-NVT', 'GCMC'
+        RunSteps : int; must be an integer greater than zero.
+            Sets the total number of simulation steps.
+        Temperature : Temperature of system in Kelvin (K)
+
+        input_variables_dict: dict, default = None
+            These input variables are optional and override the default settings.
+            Changing these variables likely required for more advanced systems.
+
+            The details of the acceptable input variables for the selected
+            ensembles can be found by running this python workbook,
+
+                print_valid_ensemble_input_variables('GCMC', description = True)
+
+            which prints the input_variables with their subsection description
+            for the selected 'GCMC' ensemble (other ensembles can be set as well).
+
+            Example : input_variables_dict = {'Restart' : False, 'PRNG' : 123,
+                                              'ParaTypeCHARMM' : True }
+
+        Returns
+        -------
+        None
+
+        Notes
+        -------
+        The details of the required inputs for the selected
+        ensembles can be found by running this python workbook,
+
+             print_valid_required_input_variables('NVT', description = True)
```

Comment:
since this is a python command to be executed in some interpreter, these lines should be prefaced with `>>>` to signify that.

see section 15 in the numpydoc style guide
https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
fixed as requested